force-app/main/default/classes/RD2_RecurringDonation.cls

```diff
@@ -172,8 +172,8 @@ public inherited sharing class RD2_RecurringDonation {
     */
     public Decimal getAnnualValue(){
         Decimal yearlyFrequency = RD_Constants.PERIOD_TO_YEARLY_FREQUENCY.get(rd.npe03__Installment_Period__c);
-        Decimal installmentFrequency = rd.InstallmentFrequency__c;
-        return rd.npe03__Amount__c * (yearlyFrequency / installmentFrequency);
+        Decimal annualValueToRound = rd.npe03__Amount__c * (yearlyFrequency / rd.InstallmentFrequency__c);
```

Comment:
Technically this doesn't calculate an accurate YearlyValue because it doesn't account for future changes to the schedule such a Pause or just a changed future amount or period. However, it works as a general proxy to determine if the value of the RD went up or down based on a change to something. That noted, it might be useful to update the description on the method to explain usage given that it's not calculating the same value as the "Current Yearly Value" or "Next Yearly Value" fields on the RD.

Reply:
Sure, I'll make sure the fields have an accurate description as well.